src/styleguide/_patterns/01-atoms/07-forms/00-text-input.njk

```diff
@@ -0,0 +1,12 @@
+{# Reusable macro for other templates to import: #}
+{%- macro textInput(id, classNames, type, disabled) -%}
```

Comment:
Maybe provide some default values for `type` (probably `"text"`) and `disabled` (probably `false`) for convenience?

Reply:
Added default type of text. The disabled value should default to nothing (false) or "disabled" if true is passed.